subsys/testsuite/ztest/Kconfig

```diff
@@ -181,6 +181,58 @@ config TEST_LOGGING_FLUSH_AFTER_TEST
 	default y
 	depends on MULTITHREADING
 
+config ZTEST_MAIN
```

Comment:
why do we need this? Either you run with shell, then you jump into shell and run the tests from there, or run the tests without shell and just execute the test as usual. Do not see the value of this.